firmware/src/serial/serial_protocol_plaintext.cpp

```diff
@@ -30,22 +30,22 @@ void SerialProtocolPlaintext::loop()
     while (stream_.available() > 0)
     {
         int b = stream_.read();
-        if (b == 0)
```

Comment:
given that this file is not enabled now (switch is commented) you can undo changes in this file